infrastructure/modules/ansible/roles/nimona_bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Create and start nimona bootstrap container
+  community.general.docker_container:
+    name: bootstrap
+    image: "ghcr.io/nimona/nimona:{{ nimona_bootstrap_version }}"
+    pull: yes
+    recreate: '{{ nimona_bootstrap_version == "latest" }}'
+    restart_policy: always
+    state: started
+    container_default_behavior: no_defaults
+    privileged: yes
+    networks_cli_compatible: yes
+    env:
+      NIMONA_LOG_LEVEL: "info"
```

Comment:
This might be a bit overwhelming for the disk :p might be ok for staging but prod is a bit too much.

Reply:
What should we set it to on production then?